ios/SitumPlugin.m

```diff
@@ -711,6 +711,12 @@ - (dispatch_queue_t)methodQueue
     }
 }
 
+RCT_EXPORT_METHOD(getDeviceId:(RCTResponseSenderBlock)callbackBlock)
+{
+    callbackBlock(@[@{@"deviceId": SITServices.deviceID}]);
```

Comment:
Hi Noman, why is this method returning an array and the one in java is returning a map? It is due to some React limitation?

Reply:
Hi Adrian, yeah its RN limitation. 

If you look at all other responses, we wrap all dictionionaries in array when passing to callback.

Reply:
Sorry, I didn't realize that. Thanks for the answer! I will merge this PR then.